Quest Behaviors/ForcedMount.cs

```diff
@@ -110,12 +110,19 @@ public ForcedMount(Dictionary<string, string> args)
 
             else
             {
-                Mount.FlyingMounts.First().CreatureSpell.Cast();
+                if (!string.IsNullOrEmpty(Styx.Helpers.CharacterSettings.Instance.FlyingMountName))
+                {
+                    foreach (var mount in Mount.FlyingMounts.Where(mount => mount.Name == Styx.Helpers.CharacterSettings.Instance.FlyingMountName))
+                        Mount.FlyingMounts.FirstOrDefault(m => m.CreatureSpellId == mount.CreatureSpellId)?.CreatureSpell.Cast();
+                }
+                else
+                    Mount.FlyingMounts.First().CreatureSpell.Cast();

```

Comment:
Indeed, Flightor.MountHelper.MountUp can be used here. It handles shapeshifts, preferred mounts etc. already.